ui/packages/consul-ui/app/components/topology-metrics/source-type/index.hbs

```diff
@@ -0,0 +1,6 @@
+<span
+  class="topology-metrics-source-type"
+  {{tooltip "This connection was defined in a proxy registration."}}
```

Comment:
Here I need the `tooltip` modifier but I can't use the modifier with translations. LMK if you know how to make both work. @johncowen

Reply:
You can nest helpers within things in handlebars with round brackets:

```hbs
{{!-- modifiers --}}
{{tooltip (t "the.identifier")}}

{{!-- block helpers --}}
{{#let (t "the.identifier") as |text|}}{{/let}}

{{!-- inline helpers --}}
{{another-helper (t "the.identifier")}}
```

https://guides.emberjs.com/v3.22.0/components/helper-functions/#toc_nested-helpers